examples/geometry/volumes/distance/distancetransform2D.cpp

```diff
@@ -142,14 +144,21 @@ int main()
   Display2DFactory::drawImage<HueTwice>(board, dtL1, 0.0, maxv1 + 1);
   board.saveSVG ( "example-DT-L1.svg" );
 
+  //Explicit export with ticked colormap
   //We compute the maximum DT value on the L2 map
-  for ( unsigned int j=0;j<33;j++)
-    {
-      for(unsigned int i=0; i<33; i++)
-        trace.info()<< dtL2(Z2i::Point(i,j)) << " ";
-      trace.info()<<std::endl;
-    }
-
+  board.clear();
+  TickedColorMap<double, GradientColorMap<double> > ticked(0.0,maxv2);
```

Comment:
Since it is an example, I suggest to add explicitly the White color as 3rd parameter.